modules/ROOT/pages/managing-applications-and-domains.adoc

```diff
@@ -4,12 +4,12 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: agent, mule, esb, servers, monitor, notifications, external systems, third party, get status, metrics
 
-image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
-image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
-image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
+image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
```

Comment:
After having played around with amc_setup (see https://docs.mulesoft.com/runtime-manager/installing-and-configuring-runtime-manager-agent#amc_setup-parameters), it seemed to me that we cannot configure the agent with both the -H and the -I argument for example. So I am not sure how we can achieve using the agent API for domain deployments and connectivity with the Anypoint platform at the same time. I am sure that @mcocaro can clarify easily though if this section applies to Hybrid and PCE as well.

Reply:
-H and -I are mutually exclusive

Reply:
@bauerchristina - is the issue that the icons should be on specific headings within the topic? Or that we need more clarification that -I/-H are mutually exclusive? Or both?

Reply:
Added icons to the install subheads and a link from this topic as requested.